modules/fileout_netcdf/FONcArray.cc

```diff
@@ -668,6 +668,32 @@ void FONcArray::write_nc_variable(int ncid, nc_type var_type) {
     else
         d_a->intern_data(*get_eval(), *get_dds());
 
+    bool d_io_flag = d_a->get_dio_flag();
+    if (d_io_flag)
+        BESDEBUG("fonc", "d_io_flag is true"<<endl);
+    else
+        BESDEBUG("fonc", "d_io_flag is false"<<endl);
+    
+    if (d_io_flag) {
+        Array::var_storage_info dmrpp_vs_info = d_a->get_var_storage_info();
+        for (const auto& def_lev:dmrpp_vs_info.deflate_levels) 
+            BESDEBUG("fonc", "deflate level: "<<def_lev<<endl);
```

Comment:
For these for loops, it is probably better to wrap the loop with #ifndef NDEBUG ... #endif. The code will only be compiled when NDEBUG is not defined. The double negative logic is annoying, but using NDEBUG is used by the assert.h header, etc., so I used it without changing its definition.

Reply:
Done